api_handler/api_calls.py

```diff
@@ -37,8 +37,6 @@ def check_if_response_successful(self, response: Response) -> bool:
                 logger.error('JSONDecodeError encountered')
                 response_successful = False
 
-        if not response_successful:
-            logger.error(response.text)
```

Comment:
This statement was just printing out the JSON error response text, which is just the big JSON text returned. So this statement create noise in the log, hence deleting it